rest_tools/server/handler.py

```diff
@@ -160,6 +170,11 @@
     @wtt.evented()
     def prepare(self):
         """Prepare before http-method request handlers."""
+        logger.debug(
+            f"{self.request.path} {self.request.method}("
+            f"{self.path_args}, {self.path_kwargs}) [{self.__class__.__name__}]"
```

Comment:
It is technically correct, since the path is user supplied and may not be constrained.  It's usually fine, but because the regex may be open-ended, this may not be ideal in the generic handler.